src/http/server/response.cr

```diff
@@ -255,7 +255,9 @@ class HTTP::Server
       private def unbuffered_write(slice : Bytes) : Nil
         return if slice.empty?
 
-        unless response.wrote_headers?
+        if response.headers.includes_word?("transfer-encoding", "chunked")
```

Comment:
**nitpick:** We're using upper cased header names everywhere else in the codebase, so we should follow here as well.
```suggestion
        if response.headers.includes_word?("Transfer-Encoding", "chunked")
```
Ditto below.

Reply:
**thought:** Also I'm not sure about `includes_word?`. If the header is set to `Transfer-Encoding: chunked, gzip`, I don't think we can really deal with that. So it might be better to look only for `Transfer-Encoding: chunked` exactly.

Reply:
Arguably would be better to use lower case everywhere, but that's for another discussion.

Pushed up another commit 👍

Reply:
What the spec says about such situation?

Reply:
The RFC for HTTP/1.1 calls out in https://www.rfc-editor.org/rfc/rfc9110#name-field-names: 

> Field names are case-insensitive

Whereas the RFC for HTTP/2 calls out in https://datatracker.ietf.org/doc/html/rfc9113#section-8.2:

> Field names MUST be converted to lowercase when constructing an HTTP/2 message.

So to me this means that using lowercase is always preferable so it's compatible with both.

Reply:
I was referring to the case with compound values.